package.json

```diff
@@ -14,7 +14,8 @@
     "predev": "npm-run-all generate build",
     "dev": "npm-run-all --parallel start watch",
     "pretest": "npm-run-all generate:*",
-    "test": "jest",
+    "test": "yarn test:only",
```

Comment:
why this change?

Reply:
`yarn test` perfoms code generation before actually running test. This is very unpleasant when working with a short dev-test-refactor cycle. So I introduced a `yarn test:only` script that basically doesn't trigger `pretest`.